.github/workflows/build-pull-request.yml

```diff
@@ -3,6 +3,8 @@ on:
   pull_request:
     branches:
       - master
+      - 'integration/**'
```

Comment:
Temporary added until we release version 6.0.0, also some features will depend on some others, will remove later.